README.md

```diff
@@ -261,6 +264,26 @@ As a rule, it is best to use percentages for table column widths.
 
 If you have tables with a mix of percentage and absolute values for column widths and you have cells that span columns where the widths involved are mixed, Wordinator issues a warning message. The resulting table will likely not be correct.
 
+### Vertical (Row) Spans in Tables
+
+Wordinator supports vertical (row) spanning but requires that every cell in the vertical span be accounted for using `<vspan/>` markers in the `<td>` elements. 
+
+Thus, if a cell in row 1 specifies rowspan of 3, the next two rows must have `<td><vspan/></td>` elements in that column.
+
+Likewise, if the cells are also horizontally spanned, each placeholder cell must specify the same `colspan` value as the first cell that specifies `rowspan`.
+
+### Using MathML
+
+If your SimpleWPML document includes MathML markup (see the simplewpml-mathml3.rng grammar), then Wordinator will attempt to use the XSLT transform `MML2OMML.XSL` to transform the MathML into OOML markup before adding it to the DOCX.
+
+The `MML2OMML.xsl` transform is not open-source and so cannot be included in the release package. However, you should be able to find it in any Microsoft Office distribution, i.e.:
+
+```
+/Applications/Microsoft Office 2011/Microsoft Word.app/Contents/Resources/mathml2omml.xsl
+```
+
+To make the transform available to Wordinator, name it "MML2OMML.XSL" and put it in a directory on the Java class path.
```

Comment:
Putting the XSL file on the classpath is not going to work if wordinator is invoked with `-jar wordinator.jar`, as Java will then ignore the classpath. That's why I had some extra text about running it with `-cp` etc.